pyproject.toml

```diff
@@ -2,5 +2,121 @@
 requires = ["setuptools>=64", "setuptools_scm[toml]>=6.2", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+description = "One line description of your module"
+dependencies = []
+dynamic = ["version"]
+license = { file = "LICENSE" }
+name = "python3-pip-skeleton"
+readme = "README.rst"
+requires-python = ">=3.8"
+
+[project.optional-dependencies]
+dev = [
+    "black",
+    "isort",
+    "mypy",
+    "flake8",
+    "flake8-isort",
+    "flake8-pyi",
+    "Flake8-pyproject",
+    "pipdeptree",
+    "pre-commit",
+    "pydata-sphinx-theme",
+    "pytest-cov",
+    "setuptools_scm[toml]>=6.2",
+    "sphinx-autobuild",
+    "sphinx-copybutton",
+    "sphinx-design",
+    "tox",
+    "tox-direct",
+    "types-mock",
+]
+
+[project.scripts]
+python3-pip-skeleton = "python3_pip_skeleton.__main__:main"
+
+[project.urls]
+GitHub = "https://github.com/DiamondLightSource/python3-pip-skeleton"
+
+[[project.authors]]
```

Comment:
Is there a good reference for writing pyproject.toml files? A quick google didn't tell me how to add multiple maintainers here, or what the double square brackets meant

Reply:
[PEP621](https://peps.python.org/pep-0621/) has worked for me thus far

Reply:
Coming from YAML, this is arcane.

So to add multiple authors we duplicate the ``[[project.authors]]` section? I guess we should put a comment to that effect here...

Reply:
I really don't understand the choice of TOML / the schema chosen, but here we are. I've added a brief comment about this